boondoggle/cli.py

```diff
@@ -42,7 +42,9 @@ def cli(context, profile, region):
               help="Path to a cloudformation file.")
 @click.option('--outputs-from',
               help="Use another stack's outputs as inputs to this one.")
-def ensure(context, name, params, url, file, outputs_from):
+@click.option('--useprevious',
```

Comment:
Look here: http://click.pocoo.org/5/options/#boolean-flags
```
@click.option('--shout/--no-shout', default=False)
```

Preserving the same semantics, we could do:
```
@click.option('--preserve/--no-preserve', default=False)
```

Or something. This resolves to a boolean